atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/timestamp/CleanTransactionRange.java

```diff
@@ -76,7 +74,7 @@ protected int executeTimestampCommand(AtlasDbServices services) {
             }
 
             long commitTs = TransactionConstants.getTimestampForValue(value.getContents());
-            if (commitTs <= timestamp) {
+            if (commitTs <= backupTimestamp) {
```

Comment:
I don't even think this should be called `backupTimestamp`. The CLI is to do with cleaning the transaction table and doesn't care whether it's for backup or w/e.